### ISSUE TITLE
`#` headers being stripped from README

### ISSUE THREAD

**Issue by ursi** [purescript/pursuit#427]
I just realized the `#` headers on my README are not being displayed on pursuit.
[Pursuit](https://pursuit.purescript.org/packages/purescript-point-free/0.1.1)
[GitHub](https://github.com/ursi/purescript-point-free/tree/v0.1.1)

**Comment by hdgarrood** [comment]
Yes, this is by design. Usually the h1 headings, ie those introduced by a single #, contain the library name; we remove them in Pursuit because otherwise it would be duplicated. You shouldn’t really have more than one of those per file. You can fix this by just demoting each heading: each # should become ##, each ## should become ###, and so on.

**Comment by ursi** [comment]
I see, is this behaviour documented anywhere?

**Comment by hdgarrood** [comment]
It’s not, unfortunately. There’s a help page for package authors here: https://pursuit.purescript.org/help/authors which would be the place to put it.
